caikit_nlp/modules/text_embedding/embedding.py

```diff
@@ -1039,6 +1041,8 @@ def _get_tokenized(self, texts):
         the fast tokenizer with different truncation settings.
         """
 
+        pad_to_max_length = kwargs.pop("pad_to_max_length", None)
```

Comment:
Question: is there a reason to accept `pad_to_max_length` as value here instead of just `max_length` which can then be directly passed to `tokenizer` function ? 

https://huggingface.co/docs/transformers/en/main_classes/tokenizer#transformers.PreTrainedTokenizer.__call__.padding

Reply:
@gkumbhat we can change the variable name based on your suggestion. The goal is to make sure that this variable will set the tokenizer to pad to max sequence length of the model.